src/nvidia-cuda/install.sh

```diff
@@ -33,7 +33,9 @@ check_packages() {
     fi
 }
 
-check_packages wget ca-certificates
+export DEBIAN_FRONTEND=noninteractive
+
+check_packages wget ca-certificates liburcu6
```

Comment:
Looking at the test failures, looks like we need to install `liburcu6` based on the distro type. 

Also, I am seeing similar errors on the `main` branch. This PR does not impact the test failures, hence, I can track them in another issue. 

>  looks like we need to install `liburcu6` based on the distro type. 

Can you help revert the `liburcu6` change or feel free to help us with ^

Reply:
@samruddhikhandale Will do!

Reply:
@samruddhikhandale Done in 2d56c78